bve/src/parse/mesh/instructions/execution.rs

```diff
@@ -290,6 +295,14 @@ impl Executable for SetDecalTransparentColor {
     }
 }
 
+/// Actually execute the instructions provided.
+///
+/// Errors are taken from [`InstructionList::errors`] and any new ones encountered are appended put in
+/// [`ParsedStaticObject::errors`].
```

Comment:
This isn't very clear and I don't like it

It feels like you actually want to flat_map `generate_meshes` over `Result<InstructionList, Vec<String>>` to get `Result<ParsedStaticObject, Vec<String>>` (or whatever your error type is). Is there any reason you're not using Results here?

If there's errors in the instruction list, do you still try to execute the instructions? Do you check the errors to see if the instruction list can be executed in the first place? I have a lot of questions and `Result` answers most of them

Reply:
We discussed this in discord but to reiterate:

I must try to deal with really terrible code, so an error doesn't mean failure of parsing, only failure of constructing/executing that instruction.

> If there's errors in the instruction list, do you still try to execute the instructions?

Yes, I need to form an as-complete mesh as possible.

> Do you check the errors to see if the instruction list can be executed in the first place?

It can always be executed, the output might be garbage though :stuck_out_tongue:

Reply:
Alright, you're dealing with an old weird system. I'd just make that a bit more clear in the docs